### DISCUSSION TITLE
Unable to browse the Test URL

### DISCUSSION THREAD

**Discussion by eduard-un** [statamic/cms#6081] [Q&A] [answered]
Hello,

This is my first time trying to install and use Statamic.

After installing the composer, I managed to install the Statamic CLI properly.

After making edits to the .zrc file, I was able to finally run the command `statamic new $project_name` into my terminal. All went great:
![Screenshoot - iTerm2 - ed@MacBook-Air~static-sites](https://user-images.githubusercontent.com/1719735/170087805-c81a61b3-fdda-4dae-96d1-34557b374e7e.jpg)

My issue is that when I put `http://my-test-website.test/` into my browser, I get 
![Screenshoot - Google Chrome - my-test-website test](https://user-images.githubusercontent.com/1719735/170087950-e862c2a1-41f5-4b8f-9a2b-3b2a1dad8967.jpg)

I have found a thread where it said to check the `.env` file; however, not sure what to check for? This is my `.env` file: 
![Screenshoot - Code -  env — static-divi-website](https://user-images.githubusercontent.com/1719735/170088114-d68ae80c-849d-40e5-a17b-75a92fe29395.jpg)

Does this require having localhost? I usually use MAMP, I already tried that, but I cannot make it work. 

I'm feeling like a complete noob. Anyone that can provide some help would be much appreciated.

**Comment by jackmcdade** [comment, marked as answer]
Statamic does require a local dev environment. There are lots of options (MAMP being one of them, but a bit more “dated” of an approach) — our favorite is one called Laravel Valet. There’s a great tutorial series that walks you through how to get it set up, check it out: https://statamictutorials.com/course/setup-your-mac/episode/9

**Comment by eduard-un** [comment]
Hey Jack 👋 

What do you know, following the link you have shared, I managed to get it working (too bad that information was not also shared on the YouTube videos I have watched): 
![Screenshoot - Google Chrome - Home](https://user-images.githubusercontent.com/1719735/170286398-337bcf67-7dc1-4be5-8a88-784e14372e14.jpg)

PS: Mamp is great for spinning up websites that uses WordPress for example - which is what I'm using most of the time 😄 

Thank you so much for your time.

**Reply by jackmcdade** [reply]
For sure!
